From 30c715466d513d366ae131e16c0c95c18f8aac5c Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Tue, 17 Jul 2012 14:55:39 +0200
Subject: [PATCH] Fix several typos 5 of Debian series 7

This patch fixes several typos in the manual of pstoimg
Author: Roland Stigge <stigge@antcom.de>
---
 pstoimg.pin |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pstoimg.pin b/pstoimg.pin
index c5de1a4..b9e5014 100755
--- a/pstoimg.pin
+++ b/pstoimg.pin
@@ -446,13 +446,13 @@ banner() unless($opt{quiet});
 
 Use Ghostscript's anti-aliasing feature for rendering "softer" images.
 This applies to lines and edges of polygonal and oval or circular shapes.
-Only valid if Ghostscipt 4.03 or higher is installed.
+Only valid if Ghostscript 4.03 or higher is installed.
 
 =item B<-aaliastext>
 
 Use Ghostscript's anti-aliasing feature for "smoother" font characters,
 without the jagged edges. Similar to B<-antialias> for graphic components.
-Only valid if Ghostscipt 4.03 or higher is installed.
+Only valid if Ghostscript 4.03 or higher is installed.
 
 =item B<-center> I<num>
 
@@ -906,7 +906,7 @@ my $Brx = '-?\d+(?:\.\d*|)';
 =head1 DESCRIPTION
 
 B<pstoimg> iterates over the given input files and runs them through
-Ghostscipt. The resulting pnm (portable anymap files) are processed
+Ghostscript. The resulting pnm (portable anymap files) are processed
 with different Netpbm tools (cropping, color mapping, aligning, ...)
 and finally converted into (currently) either GIF or PNG format. The
 bitmaps can now be included e.g. in WWW pages.
-- 
1.7.10.3

